### PR TITLE
Feature/style result

### PR DESCRIPTION
**Pull request by AurumValian**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Testing - building tests
- [ ] Formatting - no new features
- [ ] Styling- no new features
### Detailed Description (Why is this change required? What problem does it solve? Any challenges that arose while implementing this feature? If so, how were the addressed?)

##IMPORTANT
Don't merge this PR until after Nick's is merged.

This PR adds logic to the survey to allow for reselection, putting a submit button on the last page, and options for how the BE data needs to be passed.
Currently the last page won't show up; that's because the query isn't done. Once it is, the last page will have the data.

###### Screenshots (if appropriate):

Screen Shot 2020-10-17 at 8 20 00 AM

###### Questions:

### CODE REVIEW

**Review by ejdelsztejn — APPROVED** [review]
Looks great!

File: code/web/src/modules/survey/api/actions.js [hunk 1]
```diff
@@ -62,27 +58,21 @@ export const moveBackward = () => (
...
+    }))
+      .then(response => {
+        dispatch({
+          type: SUBMIT_SURVEY,
+          style: response.data.data.style
+        })
+      })
+      .catch(error => console.error('There\'s an error'))
+    }
+)
```
Comment (Chulstro) [inline code/web/src/modules/survey/api/actions.js:76]: Dispatch is looking good!

**Review by Chulstro — APPROVED** [review]
Thanks for getting that post through!

**Comment by nickhartdev** [comment]
Really appreciate you doing this Ben! This looks great!

**Review by nickhartdev — APPROVED** [review]
Really appreciate you doing this Ben! This looks great!

File: code/web/src/modules/survey/Survey.js [hunk 2]
```diff
@@ -21,35 +21,54 @@ class Survey extends PureComponent {
...
+
+  getResults() {
+    console.log(this.props.surveyInfo.selectedClothing)
+    const styleValues = Object.values(this.props.surveyInfo.selectedClothing).reduce((styleValue, clothingStyle) => {
+      styleValue[clothingStyle] ? styleValue[clothingStyle] += 1 : styleValue[clothingStyle] = 1;
+      return styleValue;
+    }, {})
+    //If we're returning total style ids:
+    this.props.submitSurvey(styleValues)
+    //If we're returning a single style id:
```
Comment (amykarnaze) [inline code/web/src/modules/survey/Survey.js:65]: Nice job including multiple options!

**Review by amykarnaze — APPROVED** [review]
I like how you used closest to get the image a user selects.

**Pull request merged**